### ISSUE TITLE
Handle `isSubmitting` & icons

### ISSUE THREAD

**Issue by Myzel394** [kgnugur/formik-chakra-ui#104]
It would be cool if you could add left and right icons and if the inputs were disabled when `isSubmitting = true`.

I can create a PR if that's okay

**Comment by Coystark** [comment]
Any news about it? I can create the pr too.

**Comment by Myzel394** [comment]
@Coystark Yeah that would be great, since I haven't used this library for a long time now xD

**Comment by ogheneovo12** [comment]
created a pr to address the icon issue here @  #155 , take a look at it and send ur takes
